apps/chat/src/components/Marketplace/ApplicationDetails/ApplicationHeader.tsx

```diff
@@ -67,20 +65,13 @@ export const ApplicationDetailsHeader = ({ entity, onClose }: Props) => {
 
   return (
     <header className="flex gap-2 p-4 md:gap-4 md:px-6">
-      <div
-        className={classNames(
-          'flex shrink-0 overflow-hidden rounded-full bg-controls-permanent',
-          isSmallScreen() ? 'size-12' : 'size-24',
-        )}
-      >
-        <ModelIcon
-          enableShrinking
-          isCustomTooltip
-          entity={entity}
-          entityId={entity.id}
-          size={isSmallScreen() ? 48 : 96}
-        />
-      </div>
+      <ModelIcon
+        enableShrinking
+        isCustomTooltip
+        entity={entity}
+        entityId={entity.id}
+        size={isSmallScreen() ? 48 : 96}
```

Comment:
Will have same problem as in cards when user resizes browser window. There is already initialized resize observer in the top level Marketplace component with isMobile flag which can be reused here.